platform/android/app/build.gradle

```diff
@@ -159,6 +159,7 @@ dependencies {
     //noinspection FragmentGradleConfiguration
     androidTestImplementation 'androidx.fragment:fragment-testing:1.2.5'
     androidTestImplementation 'androidx.test.espresso:espresso-web:3.3.0'
+    androidTestUtil('androidx.test:orchestrator:1.4.0')
```

Comment:
Is this actually being used? Can't tell :D

Reply:
Probably not. Will check.